Adding the ability to change the view direction without moving via movement keys.

Akronymus-git opened the pull request:
# Pull Request Template

### Purpose
Adding the requested functionality of #2779

Akronymus-git committed:
Merge pull request #1 from unitystation/develop

sync

Akronymus-git committed:
Adding ability to change facing with the movement keys without moving, by using ctrl.

Akronymus-git committed:
Removing unnecessary Comment.

daalta reviewed (commented):
Looks OK! Thank you for implementing this.

How does this interact with the player being a ghost or downed?

Akronymus-git committed:
I cleaned up the code a bit and moved some functionality to their own functions.

The FacingDirection currently does not interact with the downed state.

TODO: Add documentation about it in the controls.
